submissions/jeffersonMormah/CSS/style.css

```diff
@@ -0,0 +1,219 @@
+.nav-links{
+    width: 100%;
+    margin: auto;
+    overflow: hidden;
+    display: flex;     
+}
+ul {
+    padding: 0;
+    margin: 0;    
+  }
+header {
+    background:rgb(250,128,114);
+    color: #F5F5DC;
+    padding-top: 10px;
+    min-height: 15%;
+    border-bottom:4px solid #0000FF;
+}
+header a {
+    color: #F5F5DC;
+    font-size: 17px;
+    font-family: 'Franklin Gothic Medium';
+    font-weight: bold;
+    margin-left: 320px;
+}
+header li {
+    display: inline;
+    padding: 0 30px;
+}
+header #nav-head {
+    float: left;
+    padding-bottom: 12px;
+}
+header #nav-head h2,header #nav-head p{
+    margin:0;
+    padding:0;
+    color:white;
+}
+header nav{
+    float: right;
+    margin-top: 15px;
+}
+.signup-content {
+    overflow: hidden;
+    padding: 40px auto;
+    margin: auto;
+    width: 90%;
+}
+main {
+    float: left;
+    text-align: left;
+    width: 45%;
+    margin-top: 55px;
+}
+main h1 {
+    font-size: 50px;
+    font-family: 'Franklin Gothic Medium';
+    color: coral;
+}
+main p  {
+    font-size: 25px;
+    font-family: 'Franklin Gothic Medium';
+    color: brown;
+}
+main a {
+    font-size: 20px;
+    font-family:'Franklin Gothic Medium';
+    color: blue;
+}
+aside {
+    float: left;
+    width: 48%;
+    margin-top: 55px;
+    margin-left: 40px;  
+}
+aside .form-document {
+    padding: 20px;
+    width: 94%;
+    border: 1px solid black;
+    border-radius: 20px;
+    margin-bottom: 12px;
+}
+.form-document label {
+    color: rgb(250,128,114);
+    font-size: 20px;
+    margin-bottom: 20px;
+    display: inline-block;
+    width: 48%;
+    margin-left: 20px;
+    font-family: 'Franklin Gothic Medium';
```

Comment:
@Jeffbuchi Why not set the family for the whole page instead of declaring it at every point needed.